layouts/shortcodes/card.html

```diff
@@ -1,49 +1,44 @@
-<div class="td-card card mb-4">
-  {{ with $.Get "header" }}
-    <div class="card-header">
-      {{ if eq $.Page.File.Ext "md" }}
-        {{ $.Get "header" | markdownify }}
-      {{ else }}
-        {{ $.Get "header" | htmlUnescape | safeHTML }}
-      {{ end }}
+{{/* Check parameter type */ -}}
+{{ with .Get "code" -}}
+  {{ if ne ( printf "%T" . ) "bool" -}}
+    {{ errorf "shortcode card: parameter 'code' must be either true or false" -}}
+  {{ end -}}
+{{ end -}}
+
+{{ $code := default false (.Get "code") -}}
+{{ $lang := default "" (.Get "lang") -}}
+{{ $highlight := default "" (.Get "highlight") -}}
+
+<div class="td-card card border me-4">
+{{ with $.Get "header" -}}
+    <div class="card-header{{- cond $code " bg-white" "" -}}">
```

Comment:
Here's another place where we could avoid hard-coding custom styling, but for now I'm ok if we leave this as is, and address it later.

Reply:
> Here's another place where we could avoid hard-coding custom styling,

I agree.

> but for now I'm ok if we leave this as is, and address it later.

I will open a new issue that addresses this issue shortly.